zerver/lib/validator.py

```diff
@@ -38,6 +38,14 @@ def check_string(var_name, val):
         return _('%s is not a string') % (var_name,)
     return None
 
+def check_short_string(var_name, val):
+    # type: (str, Any) -> Optional[str]
+    max_length = 200
```

Comment:
I think we should only validate short text through this function so it should be alright to keep this variable here.